test/modules/date.spec.ts

```diff
@@ -801,22 +801,22 @@ describe('date', () => {
       faker.setDefaultRefDate(() => new Date(Date.UTC(2020, 0, 1)));
       faker.seed(20200101);
       const date = faker.date.past();
-      expect(date).toEqual(new Date('2019-02-25T21:52:41.824Z'));
+      expect(date).toMatchInlineSnapshot('2019-02-25T21:52:41.819Z');
```

Comment:
I might have oversee something, but through these `toMatchInlineSnapshot` changes, we lost the check whether date is a string or actually a Date instance

Reply:
That's not what the test is about.

Reply:
Then we can simply extend the test:
```suggestion
      expect(date).toBeInstanceOf(Date);
      expect(date).toMatchInlineSnapshot('2019-02-25T21:52:41.819Z');
```